spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/DefaultSchedulerService.java

```diff
@@ -205,11 +207,25 @@ public void schedule(String scheduleName, String taskDefinitionName, Map<String,
 		ScheduleRequest scheduleRequest = new ScheduleRequest(revisedDefinition, taskDeploymentProperties,
 				deployerDeploymentProperties, revisedCommandLineArgs, scheduleName, getTaskLauncherResource());
 		Launcher launcher = getDefaultLauncher();
+		validateScheduleNameForPlatform(launcher.getType(), revisedDefinition.getName());
 		launcher.getScheduler().schedule(scheduleRequest);
 		this.auditRecordService.populateAndSaveAuditRecordUsingMapData(AuditOperationType.SCHEDULE, AuditActionType.CREATE,
 				scheduleRequest.getScheduleName(), this.auditServiceUtils.convertScheduleRequestToAuditData(scheduleRequest),
 				taskPlatform.getName());
 	}
+
+	private void validateScheduleNameForPlatform(String type, String scheduleName) {
+		if(type.equals(KUBERNETES)) {
```

Comment:
I thought we were planning to convert the case of the schedule name to be lower case and add the corresponding case insensitive checks on task definition names in other places that involves the task name references.

Reply:
I will lowercase the task-definition-name portion of the schedule name as well.  We do this when we create the pods.